packages/govuk-frontend-review/src/views/full-page-examples/rebrand-examples/index.njk

```diff
@@ -0,0 +1,360 @@
+---
+title: Child maintenance
+name: Child maintenance (Rebrand example)
+scenario: |
```

Comment:
I don't know where this is used. But I assume it would need to be rewritten for this example?

Reply:
Scenarios are used on the [full page examples overview page](https://govuk-frontend-pr-5864.herokuapp.com/full-page-examples): https://github.com/alphagov/govuk-frontend/blob/1bc5abe1e78a48f3672e740b65579084d32431bd/packages/govuk-frontend-review/src/views/full-page-examples/index.njk#L43-L52